generators/generator-microsoft-bot-conversational-core/generators/app/index.js

```diff
@@ -2,45 +2,49 @@
 // Licensed under the MIT License.
 
 'use strict';
-const Generator = require('yeoman-generator');
 
-module.exports = class extends Generator {
-  constructor(args, opts) {
-    super(args, opts);
+const {
+  BaseGenerator,
+  platforms,
+} = require('@microsoft/generator-microsoft-bot-adaptive');
 
-    this.argument('botName', { type: String, required: true });
-  }
+const packageReferences = {
+  [platforms.dotnet]: {
+    version: '1.0.0-preview.20210310.8ee9434',
+    packages: [
+      'Microsoft.Bot.Components.HelpAndCancel',
+      'Microsoft.Bot.Components.Welcome',
+    ],
+  },
+  [platforms.js]: {
+    version: 'latest',
+    packages: [
```

Comment:
Not happy with hardcoding these - to me, using the `npm` reference makes sense because that is how packages are fetched. Is there a plan to reconcile this at all?